report.md

```diff
@@ -0,0 +1,149 @@
+  and implement simple two-body problem validation cases (orbit element
+  conversion and Keplerian propagation).
+
+* First month fully devoted to the 3D impulsive maneuvers validation.
+
+* The goal for the second was to simplify and reformat previous implementation
+  (if required) in order to emulate poliastro's API.
+
+* During the third month, all the effort would be directed towards the
+  validation of planetary frames transformations.
+
+* Last month was set with the idea of being used as a time-margin resource in
+  case some of the previous points would take more than expected.
+
+
+## Issues encountered and alternative approaches
+
+During the first weeks of the program, it became evident that some of the
+original proposed goals would required a different approach to be accomplished:
+
+* **STK implementation.** We decided not to implement the validation against
+  this software due to its lack of Linux compatibility. The GMAT and Orekit
+  software were considered enough for the purposes of the validation work.
+
+* **Suppression of poliastro's API emulation**. Trying to replicate the logic
+  used by poliastro using the Orekit wrapper introduced an extra layer of
+  complexity within the validation implementation. Therefore, this approach was
+  deprecated in favor of direct comparison between pure Orekit API based tests
+  and poliastro's ones.
+
+* **GMAT not included in CI**. Although a solution for including this software
+  within the continuous integration workflow was devised, it was rejected
+  because its excessive complexity. Including GMAT meant downloading more than
+  300MB every time the CI was triggered and building a Python wrapper to manage
+  GMAT outputs. Therefore, only validation scripts were included as part of the
+  testing process.
+
+## Enhancements to the original proposal
+
+In addition to the initial objectives, some others were also implemented:
+
+* **Continuous integration setup**. By making use of GitHub's actions, it was
+  possible to introduced a CI workflow within the main poliastro's repository
+  which triggers the whole validation cases every time a new piece of code is
+  merged into the main branch.
+
+
+## Obtained results and benefits
+
+Once all required poliastro's features were validated, the following results
+arose:
+
+* An error was detected in the Moon's rotational elements, which
+  which introduced numerical differences of the order of 1E-5. These proof the
+  frame validation logic to be successful.
+
+* The frame logic inside poliastro's API was proof to be right. This logic could
+  be expanded to the Astropy project, so other projects might benefit from
+  this.
+
+* Auxiliary frames, such us planetary-fixed non-parallel to ICRF, can be
+  implemented.
+
+* Groundtrack plots can be now extended to the rest of planets of the Solar
+  System.
+
+
+## Associated pull-requests 
+
+All the pull-requests generated as a result of this work are listed down below,
```

Comment:
No hyphen in pull-request, pull request is fine (here and elsewhere)

Reply:
Fixing this 🔧